app/src/main/java/com/internship/firstweekapp/util/BindingAdapter.kt

```diff
@@ -0,0 +1,18 @@
+package com.internship.firstweekapp.util
+
+import android.graphics.Color
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.databinding.BindingAdapter
+
+@BindingAdapter("android:textColorAndText")
+fun textColorFromText(view: AppCompatTextView, text: String) {
+    view.text = text
+    view.setTextColor(
+        when (text) {
+            "Choose your variant!" -> Color.RED
```

Comment:
Definitely not good idea using  string checking as color condition
thinking about what happens is you localizate the app
right way - just create additional error field in viewmodel and pass it into binding